recipes/base_config.rb

```diff
@@ -58,3 +58,8 @@
   # Mount FSx
   include_recipe 'aws-parallelcluster::fsx_mount'
 end
+
+# Install EFA
+if node['cfncluster']['enable_efa'] == 'compute' && node['cfncluster']['cfn_node_type'] == 'ComputeFleet'
```

Comment:
what if node['cfncluster']['enable_efa'] is not set?

Reply:
It's always set, see https://github.com/aws/aws-parallelcluster/pull/1081#discussion_r289290570